kubernetes/resource_kubernetes_ingress_test.go

```diff
@@ -185,21 +185,6 @@ func TestAccKubernetesIngress_WaitForLoadBalancerGoogleCloud(t *testing.T) {
 	})
 }
 
-func testAccCheckKubernetesIngressForceNew(old, new *api.Ingress, wantNew bool) resource.TestCheckFunc {
```

Comment:
Not related to the original issue, just noticed that it is not in use, and since I touched this file, I cleaned it up a bit.